projects/igniteui-angular/src/lib/data-operations/filtering-expressions-tree.ts

```diff
@@ -10,15 +10,63 @@ export declare interface IFilteringExpressionsTree {
 }
 
 export class FilteringExpressionsTree implements IFilteringExpressionsTree {
+
+    /**
+     * Sets/gets the filtering operands.
+     * ```typescript
+     * const gridExpressionsTree = new FilteringExpressionsTree(FilteringLogic.And);
+     * const expression = [
+     * {
+     *   condition: IgxStringFilteringOperand.instance().condition('contains'),
+     *   fieldName: 'Column Field',
+     *   searchVal: 'Value',
+     *   ignoreCase: false
+     * }];
+     * gridExpressionsTree.filteatingOperands.push(expression);
```

Comment:
filteatingOperands

Reply:
and please add in the end how to apply the expressions to the grid:
this.grid.filteringExpressionsTree = gridExpressionsTree;